src/kernel/arch/x86/paging.zig

```diff
@@ -0,0 +1,122 @@
+const std = @import("std");
+const panic = @import("../../kmain.zig").panic;
+const arch = @import("../../arch.zig").internals;
+const isr = @import("isr.zig");
+const assert = std.debug.assert;
+const MemProfile = @import("../../mem.zig").MemProfile;
+
+const ENTRIES_PER_DIRECTORY = 1024;
+
+const PAGE_SIZE = 0x400000;
+const PAGE_SIZE_4KB = PAGE_SIZE / 1024;
+const PAGES_PER_DIR_ENTRY = PAGE_SIZE_4MB / PAGE_SIZE;
```

Comment:
Where is PAGE_SIZE_4MB defined?

Reply:
Ah, this is evidence of Zig's lazy analysis. I originally had a distinction between 4kb and 4mb paging but since I couldn't get 4kb to work I removed it. This is leftover from then.